go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/davecgh/go-spew v1.1.1
 	github.com/golang/snappy v0.0.4
 	github.com/google/go-cmp v0.6.0
-	github.com/klauspost/compress v1.13.6
+	github.com/klauspost/compress v1.17.10
```

Comment:
This version seems to require Go 1.19+.

Reply:
Ah, that's why `go mod --tidy` was updating the `go` versions!